pex/pip/version.py

```diff
@@ -113,4 +113,11 @@ def values(cls):
         requires_python=">=3.7",
     )
 
+    v23_1 = PipVersionValue(
+        version="23.1",
+        setuptools_version="67.6.1",
+        wheel_version="0.38.4",
```

Comment:
This should be 0.40.0 assuming it works: https://pypi.org/project/wheel/#history
That was released just prior to Pip 23.1 and I aim to take the latest of setuptools and wheel up to the given Pip release date barring other issues.

Reply:
ah ok! Yeah missed that.